core/src/ast/ddl.rs

```diff
@@ -102,9 +110,26 @@ impl ToSql for ColumnUniqueOption {
     }
 }
 
+impl ToSql for OperateFunctionArg {
+    fn to_sql(&self) -> String {
+        let OperateFunctionArg {
+            name,
+            data_type,
+            default,
+        } = self;
+        let default = default
+            .as_ref()
+            .map(|expr| format!(" DEFAULT {}", expr.to_sql_unquoted()))
```

Comment:
Here, we need to use `expr.to_sql` rather than `to_sql_unquoted`
ref. https://github.com/gluesql/gluesql/pull/1130